cli/src/maintenance.rs

```diff
@@ -692,6 +709,21 @@ impl SolidoState {
         None
     }
 
+    /// If there is a validator ready for removal, try to remove it.
```

Comment:
This comment is copy-pasted from elsewhere, please add an up-to-date doc comment.